Source/swiftlint/Helpers/CompilerArgumentsExtractor.swift

```diff
@@ -2,62 +2,18 @@ import Foundation
 import SourceKittenFramework
 
 struct CompilerArgumentsExtractor {
-    static func allCompilerInvocations(compilerLogs: String) -> [String] {
-        var compilerInvocations = [String]()
+    static func allCompilerInvocations(compilerLogs: String) -> [[String]] {
+        var compilerInvocations = [[String]]()
         compilerLogs.enumerateLines { line, _ in
             if let swiftcIndex = line.range(of: "swiftc ")?.upperBound, line.contains(" -module-name ") {
-                let invocation = line[swiftcIndex...]
-                    .components(separatedBy: " ")
```

Comment:
This was the cause of the bug. The rest of the changes in this PR are just refactoring.